.github/workflows/release.yml

```diff
@@ -52,7 +64,4 @@ jobs:
             --base main \
             --head release/${{ inputs.version }}
         env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        outputs:
-          target-branch: release/${{ github.event.inputs.version }}
-          run: ${{ github.event.inputs.update-language-server}}
\ No newline at end of file
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
nit: Add a newline to end of this file so this error goes away

Reply:
👍